build.sbt

```diff
@@ -43,3 +43,12 @@ lazy val gpb = project
     name := "bytes-gpb",
     libraryDependencies += "com.google.protobuf" % "protobuf-java" % "2.6.1"
   )
+
+lazy val gpbv3 = project
+  .in(file("gpb-v3"))
```

Comment:
Cannot we use here just `gpb`, and so don't duplicate the Java code? 🤔

Reply:
No, that's not possible. I agree that the code duplication is not nice but consider that this code hasn't been change in years and it probably won't be in the future so let's not overcomplicate this.